### PR TITLE
update service worker for correct scope

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: src/Photos.vue [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default {
 				navigator.serviceWorker.register(generateUrl('/apps/photos/service-worker.js', {}, {
 					noRewrite: true,
 				}), {
-					scope: '/',
+					scope: generateUrl('/apps/photos'),
```
Comment (szaimen) [inline src/Photos.vue:173]: However assets might be loaded through a different route so not sure if this would really work. Thoughts?

**Review by artonge — APPROVED** [review]

File: src/Photos.vue (hunk 1 above)
Comment (artonge) [reply src/Photos.vue:173]: Previews are now loaded from `/apps/photos/api/v1/preview` so it makes sense

Comment (artonge) [reply src/Photos.vue:173]: From the doc: the `scope` property is the "range of URLs a service worker can control". So it makes sense.

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register#parameters

Comment (szaimen) [reply src/Photos.vue:173]: all right, cool!

Comment (skjnldsv) [reply src/Photos.vue:173]: Initially we had an impact on all previews, but now that we are using a custom route for previews that only applies on photos, I guess that make sense :)

Comment (szaimen) [reply src/Photos.vue:173]: Great! Is this custom route only for NC25?

Comment (skjnldsv) [reply src/Photos.vue:173]: yep

Comment (skjnldsv) [reply src/Photos.vue:173]: https://github.com/nextcloud/photos/blob/2cad0f85b7ccb4e98735823a8ca8b014d486029c/appinfo/routes.php#L123-L129

**Comment by szaimen** [comment]
cypress failure unrelated

**Comment by szaimen** [comment]
/backport https://github.com/nextcloud/photos/commit/51cdc208ed65880d3ab3f8b41e94ac5c4f2a221a to stable25

**Pull request merged**